src/mappings/ipfs-data.ts

```diff
@@ -181,6 +180,31 @@ function getValueAsBigInt(jsonObject: TypedMap<string, JSONValue>, key: string):
   return value.toBigInt()
 }
 
+function getValueAsBoolean(jsonObject: TypedMap<string, JSONValue>, key: string): boolean {
+  const value = jsonObject.get(key)
+
+  if (value == null || value.isNull() || value.kind != JSONValueKind.BOOL) {
+    return false
+  }
+
+  return value.toBool()
+}
+
+function getValueAsArray(jsonObject: TypedMap<string, JSONValue>, key: string): string[] | null {
+  const value: JSONValue | null = jsonObject.get(key)
+
+  if (value == null || value.isNull() || value.kind != JSONValueKind.ARRAY) {
+    return null
+  }
+  let stringArray: string[] = []
+  for (let i: number = 0; i < stringArray.length; ++i) {
+    //TODO indexing error here
```

Comment:
Build error here - still not fixed

"- Compile subgraphERROR TS2329: Index signature is missing in type '~lib/@graphprotocol/graph-ts/common/value/JSONValue | null'.

     const stringValue: string = value[i].toString()
"